README.md

```diff
@@ -22,6 +22,11 @@
 * Запусти webpack (`yarn start` или `npm start`)
 * А для билда `yarn build` или `npm run build`
 
+# Белая тема (thanks to @ironsoul0!)
+Для поддержки белой темы, стоит учитывать следующие вещи:
+* При написании новых React компонентов, вы можете задавать стили для белой темы через написание стилей для `.white .yourClassName`. При этом, стили для `.yourClassName` будут применяться для темной темы. Следовательно, в JSX от вас лишь потребуется задавать prop `className={yourClassName}`.
```

Comment:
Please add a comment about adding assets and switching between them.

Reply:
Done.

Reply:
> Please add a comment about adding assets and switching between them.

Done.